_includes/v3.6/release-notes/v3.6.0-release-notes.md

```diff
@@ -0,0 +1,38 @@
+09 Mar 2019
+
+#### Calico IPAM for Kubernetes API datastore
+
+You now have full feature parity for Calico IP address management (IPAM) irrespective of the datastore you are using with your Calico deployment. This enables all of Calico’s
+IP address management features, such as multiple IP pools, per-pod, per-namespace, and per-node pool specification when using the Kubernetes API datastore
+in addition to existing etcdv3 support.
+
+Upgrading existing clusters using Calico with host-local IPAM is seamless - simply apply the v3.6 manifest and you’ll be using Calico IPAM!
+
+GitHub PRs:
+ - [libcalico-go #1010](https://github.com/projectcalico/libcalico-go/pull/1010) (@caseydavenport, @briansan)
+ - [cni-plugin #683](https://github.com/projectcalico/cni-plugin/pull/683) (@briansan)
+ - [cni-plugin #684](https://github.com/projectcalico/cni-plugin/pull/684) (@lwr20, @song-jiang)
+ - [kube-controllers #340](https://github.com/projectcalico/kube-controllers/pull/340) (@caseydavenport)
+
+#### Openstack enhancements
+
+Calico v3.6 has several enhancements for your Openstack deployments for better operational efficiency and resiliency. Now you can write Calico policy rules with
```

Comment:
Worth noting that it's extra important to follow the correct upgrade ordering this time around or the upgrade will be disruptive.